File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/issuedTicket/exception/CanNotCancelException.java
package band.gosrock.domain.domains.issuedTicket.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class CanNotCancelException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new CanNotCancelException();

    private CanNotCancelException() {
        super(IssuedTicketErrorCode.CAN_NOT_CANCEL);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/issuedTicket/exception/CanNotCancelException.kt
package band.gosrock.domain.domains.issuedTicket.exception

import band.gosrock.common.exception.DuDoongCodeException

class CanNotCancelException private constructor() : DuDoongCodeException(IssuedTicketErrorCode.CAN_NOT_CANCEL) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = CanNotCancelException()
    }
}
